contrib/wordlyworks/main.py

```diff
@@ -0,0 +1,46 @@
+"""
+Use named entity recognition to extract entities from text description (e.g. Spacy)
+Download SVGs from https://thenounproject.com/
+Ask large language model to arrange
+Generate combined SVG
+Run through e.g. Stable Diffusion and/or ask large language model to generate animation code
+"""
+
+from pprint import pformat
+from requests_oauthlib import OAuth1
+import os
+import requests
+from contrib.wordlyworks import ner.ner
+
+from dotenv import load_dotenv
+from loguru import logger
+import ner
+#from TheNounProjectAPI import API
+
+
+load_dotenv()
+
+
+def main():
+    key = os.environ["NOUN_PROJECT_KEY"]
+    secret = os.environ["NOUN_PROJECT_SECRET"]
+    logger.info(f"{key=} {secret=}")
+
+
+    auth = OAuth1(key, secret)
+    endpoint = "https://api.thenounproject.com/v2/icon/1"
+
+    response = requests.get(endpoint, auth=auth)
```

Comment:
Consult the noun project API to determine which URL to use in order to search, and search for each of the entities returned by `get_entities`

Reply:
Sounds good 👍  based on my current research I think we should use their `blacklist` endpoint. It allows for more control over their icons relatively compared to their other APIs.

The blacklist API is pretty neat if there are irrelevant or inappropriate terms. They will automatically be excluded compared to their `icons` endpoint. 

I'll have some stuff for us to chat about on Friday!